### DISCUSSION TITLE
How to get 2d Gaussian Grid areas added to 2d diagnostic output files?

### DISCUSSION THREAD

**Discussion by aschuh** [ufs-community/ufs-weather-model#2095] [Q&A] [answered]
It would be convenient to be able to generate the 2d grid areas of the Gaussian grid that UFS interpolates to, I'm having difficulty figuring out how to do that?  Looking for a  192 x 384 array of surface areas basically, either in separate file or added to 2d sfc diagnostics.  I tried adding:

# grid spec
  "dynamics", "area",      "area",      "grid_spec", "all", .false.,  "none", 2,

to diag_table but that doesn't seem to work.   Ideas?  

Second question might be how to extend back to native cubed sphere grid ... for "areas" for that?  This all has to do w/ reconstructing mass budgets for atm tracers post hoc.

**Comment by gspetro-NOAA** [comment]
Hi @aschuh ,

I've asked @jkbk2004 to take a look at this issue, but it's taking a little longer than expected. Hopefully one of us can get you on the right track soon. In the meantime, I wanted to let you know that we haven't forgotten about your question! 

Best,
Gillian Petro

**Comment by aschuh** [comment, marked as answer]
I posed this over at the GFDL FV3 github site and got a replies.  The summary was that I *should* be able to get areas from the cubed sphere grid w/ the right diag_table args, which I haven't been able to verify.  The answer for the Gaussian grid (output interpolated) was basically to do it offline post-hoc.

https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/discussions/313

This can probably be closed unless somebody can provide anything more?

**Reply by gspetro-NOAA** [reply]
Great! Glad you were able to get the info you need. I'll touch base w/ @jkbk2004 and then close if he has nothing else to add.
